app/uk/gov/hmrc/vatregisteredcompaniesapi/config/ServiceConfiguration.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.vatregisteredcompaniesapi.config
+
+import javax.inject.{Inject, Singleton}
+import play.api.Mode.Mode
+import play.api.{Configuration, Environment}
+import uk.gov.hmrc.play.config.ServicesConfig
+
+@Singleton
+class ServiceConfiguration @Inject()(override val runModeConfiguration: Configuration,
```

Comment:
I think the intention with this class was to stick all your values from your config file in here as some sort of global config class - if you're not using this class and are instead reading and storing the config parameters in the classes where you need it (as you seem to be doing), you probably don't need this class.

Reply:
sorry, I know I commented on injecting this class somewhere before!